src/content/docs/en/guides/rss.mdx

```diff
@@ -230,6 +230,23 @@ rss({
 If you'd prefer not to create your own stylesheet, you may use a premade stylesheet such as the [Pretty Feed v3 default stylesheet](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl). Download the stylesheet from GitHub and save into your project's `public/` directory.
 :::
 
+## Enabling auto-discovery of your RSS feed
+
+RSS autodiscovery, [as explained by the RSS Advisory Board](https://www.rssboard.org/rss-autodiscovery), is "a technique that makes it possible for browsers and other software to automatically find a site's RSS feed."
+
+All you need to do is add a `link` tag to your site's `head` element, like so:
+
+```html
+<link
+    rel="alternate"
+    type="application/rss+xml"
+    title="Your Site's Title"
+    href={`${Astro.site}rss.xml`}
+/>
+```
+
+After you've done this, your readers can put your site's base URL into their RSS reader to subscribe to your posts, without having to find the specific URL of your RSS feed!
```

Comment:
``````suggestion
## Enabling RSS feed auto-discovery

[RSS autodiscovery](https://www.rssboard.org/rss-autodiscovery) allows browsers and other software to automatically find a site's RSS feed from the main URL.

To enable, add a `<link>` tag with the following attributes to your site's `head` element:

```html
<link
    rel="alternate"
    type="application/rss+xml"
    title="Your Site's Title"
    href={`${Astro.site}rss.xml`}
/>
```

With this tag, readers of your blog can enter your site's base URL into their RSS reader to subscribe to your posts without needing the specific URL of your RSS feed.
``````
Just tightened up some of the wording to match our docs language style!

Reply:
This was not at all clear from my original wording, but RSS autodiscovery works from any URL that has the meta tag—even a specific post, for example. That could be good to capture in the wording. Do you have ideas about how best to word that?

Reply:
@tylermercer So, if I understand, even if one single page has this tag, then your ENTIRE RSS feed is discoverable?

I will also say that "not everything that is true" belongs in docs!  :smile: It is fine to keep the readers on a "happy path" with something that works, and not give them too many options that might only confuse them. So, I would lean towards not even saying anything else at all, if the above is enough guidance for them to be successful!

Is there a very specific, common use case where putting this in the head of your main site layout would not work, and you would absolutely want it somewhere else? If no, then this is fine as is, I think!

Reply:
I meant that if this tag is in your base layout's `head`, it doesn't matter which URL from your site the person puts in their reader—it doesn't have to be the base URL like I said in my original wording. E.g. a person could drop `https://tylermercer.net/posts/software/partial-clone-git-repo/` into their reader and it would subscribe to my sitewide RSS feed.

But, point taken! We can keep it as-is; it's clear enough.

Reply:
Ah, I get you now! Yes, I think that's extra info that takes away from the simplicity of enabling the tag. So let's get this puppy merged and helping people for the weekend!